### PR TITLE
feat: add Accordion component with chevron icon and test file

### LINKED ISSUE #46: [FEATURE] Create Accordion Component
## Description
Develop an Accordion Component. This component will be implemented in the pricing page to list all FAQs. The Accordion component should not drop down more than 1 item.

## Acceptance Criteria

- The accordion component should be implemented according to the design [Use this as guideline](https://ui.shadcn.com/docs/components/accordion)
- Ensure the accordion component is responsive in all screen sizes
- Write test for your component

### Requirements
**The prop should receive the Accordion Header, the question, and the Accordion content, the response**

### Test Script

- Ensure the component renders the requirements accordingly
- Ensure the component receives the accordion header, the question.
- Ensure the component receives the accordion content, the response.

## Tasks

- [x] Develop the component based on the requirements and design
- [x] Make component responsive on all screens
- [x] Write unit test for the component

### Link
[Accordion](https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=4-20796&m=dev)

### PR DESCRIPTION
**Pull request by Confidenceb**
# Description

**Closes #46 **

# Changes proposed

## What did you do?

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [] My code follows the code style of this project.
- [] This PR does not contain plagiarized content.
- [] The title and description of the PR is clear and explains the approach.
- [] I am making a pull request against the **dev branch** (left side).
- [] My commit messages styles matches our requested structure.
- [] My code additions will fail neither code linting checks nor unit test.
- [] I am only making changes to files I was requested to.

# Screenshots/Videos

Figma link: https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=4-20796&m=dev

Screenshot 2024-07-20 023213
Screenshot 2024-07-20 023154
Screenshot 2024-07-20 023123

### CODE REVIEW

File: src/components/Accordion.tsx
Comment (JulianaSau) [inline src/components/Accordion.tsx:1]: Shadcn already has an accordion component. why are you re-inventing the wheel?
Just install the accordion component, if it's not already there and use that

Comment (Confidenceb) [reply src/components/Accordion.tsx:1]: I should do it again using Shadcn?

Comment (JulianaSau) [reply src/components/Accordion.tsx:1]: yes

Comment (prudentbird) [reply src/components/Accordion.tsx:1]: Use lucide react chevron down

**Review by prudentbird — CHANGES_REQUESTED** [review]
Use lucide react chevron down

File: src/components/Accordion.tsx
Comment (Confidenceb) [reply src/components/Accordion.tsx:1]: Using the same code or with Shadcn?

File: src/components/Accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    content: "Answer to question 1.",
+  },
+  {
+    header: "Is there a discount for annual subscriptions?",
+    content: "Answer to question 2.",
+  },
+  { header: "Do you offer a free trial?", content: "Answer to question 3." },
+];
+
+export function AccordionDemo() {
```
Comment (JulianaSau) [inline src/components/Accordion.tsx]: name this component appropriately

File: src/components/chevron-down.svg
Comment (JulianaSau) [inline src/components/chevron-down.svg:1]: delete this. use icons from lucide-icons

**Review by JulianaSau — DISMISSED** [review]

File: src/components/chevron-down.svg
Comment (Confidenceb) [reply src/components/chevron-down.svg:1]: okay

**Review by Confidenceb — COMMENTED** [review]
Done using Shadcn

**Review by Confidenceb — COMMENTED** [review]
Done

File: src/components/chevron-down.svg
Comment (Confidenceb) [reply src/components/chevron-down.svg:1]: Deleted
Already used the Lucide-icons

**Review by Confidenceb — COMMENTED** [review]
Done

File: src/components/Accordion.tsx (hunk 1 above)
Comment (Confidenceb) [reply src/components/Accordion.tsx]: Done
Changed to 
AccordionComponent

**Review by Confidenceb — COMMENTED** [review]
done

**Review by Confidenceb — COMMENTED** [review]
done

**Review by JulianaSau — APPROVED** [review]

**Comment by JulianaSau** [comment]
@Confidenceb is your branch up to date with dev?

**Pull request closed without merging**